src/renderers/server-render.jsx

```diff
@@ -190,19 +182,15 @@ const makeRenderer = (
 	})
 	.first(state => state.preRenderChecklist.every(isReady => isReady));  // take the first ready state
 
-	console.log(JSON.stringify({
-		message: `Dispatching RENDER for ${request.url.href}`,
-		type: 'dispatch',
-		info: {
-			url: request.url,
-			method: request.method,
-			id: request.id,
-		}
-	}));
-	store.dispatch({
+	const action = {
 		type: SERVER_RENDER,
 		payload: url,
-	});
+	};
+	server.app.logger.debug(
```

Comment:
A follow up question / suggestion -- is there a way to make `server` available in most places without complicating things, so that 99% of the time we look for modules on `server.app`? (I'm thinking in the context of configs as well as logging)